### PR TITLE
rewriting the JS, readable, and fixed some issues meanwhile.

### PR DESCRIPTION
**Pull request by lolis12**

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -136,46 +142,70 @@ let time = startingMinutes * 60;
...
 function updateCountdown() {
         const minutes = Math.floor(time / 60);
         let seconds = time % 60;
         seconds = seconds < 10 ? '0' + seconds : seconds;
 
         countdown.innerHTML = `${minutes}:${seconds}`;
         time--;
-        if (minutes == 0 && seconds == 0) {
+
+        if (minutes === 0 && seconds === 0) {
```
Comment (srbmaury) [inline src/main.js:158]: This caused a problem, 'seconds' is a string and you cannot compare it like this. It ran into a never-ending negative timeout.

Comment (oriapp) [reply src/main.js:158]: true, my false. I'll push changes soon

Comment (srbmaury) [reply src/main.js:158]: I've already corrected it.

Comment (oriapp) [reply src/main.js:158]: alright, thanks (:

**Pull request merged**